pkgs/applications/science/electronics/horizon-eda/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchFromGitHub, pkgconfig, sqlite, libyamlcpp, libuuid, gnome3, epoxy, librsvg, zeromq, cppzmq, glm, libgit2, curl, boost, python3, opencascade, libzip, podofo, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+	name = "horizon-eda-${version}";
```

Comment:
Please use `pname = "horizon-eda";` instead as `name = "${pname}-${version}";` is already implied.